README.md

```diff
@@ -19,9 +19,14 @@ $ npm install -g @votingworks/hmpb-interpreter # or, with npm
 ```ts
 import { Interpreter } from '@votingworks/hmpb-interpreter'
 
-// Configure contests via an election.json. Contests on printed ballots must
-// appear in the same order they appear in this configuration.
-const interpreter = new Interpreter(election)
+const interpreter = new Interpreter({
+  // Configure contests via an election.json. Contests on printed ballots must
+  // appear in the same order they appear in this configuration.
+  election,
+
+  // Require at least 20% filled in targets.
+  markScoreVoteThreshold: 0.2,
```

Comment:
I added this as an option at this level for now, though I suspect we'll want to expose the ability to re-evaluate a given set of marks at different thresholds based on something in the UI later. Until we actually have that use case, I'm going to leave this simple setting here.